examples/advanced_operations/add_responsive_search_ad_full.py

```diff
@@ -147,9 +256,7 @@ def create_campaign(client, customer_id, campaign_budget):
     campaign_operation = client.get_type("CampaignOperation")
     campaign = campaign_operation.create
     campaign.name = f"Testing RSA via API {uuid.uuid4()}"
-    campaign.advertising_channel_type = (
-        client.enums.AdvertisingChannelTypeEnum.SEARCH
-    )
+    campaign.advertising_channel_type = client.enums.AdvertisingChannelTypeEnum.SEARCH
 
     # Recommendation: Set the campaign to PAUSED when creating it to prevent
     # the ads from immediately serving. Set to ENABLED once you've added
```

Comment:
can't comment on it directly but we should update the link below to reference the latest ref docs
https://developers.google.com/google-ads/api/reference/rpc/latest/Campaign#campaign_bidding_strategy

Reply:
The ResponsiveSearchAdInfo link further down too